### PR TITLE
Add Project Description

### PR DESCRIPTION
**Pull request by theianjones**
In this pr, I attempt to explain what this project is about.

 I have shaped work for the "Capturing and storing user information" portion of this project. This includes what technologies we are going to use and some background as to why we chose those technologies. 

training

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,38 +1,121 @@
 [![Depfu](https://badges.depfu.com/badges/3510c0fc8a9b17a045ef2389a436fe9d/count.svg)](https://depfu.com/github/theianjones/shoulditraintoday?project_id=27149)
 
-This is an opinonated Next.js starter project that makes it relatively simple to spin up a new project.
+# ![](https://res.cloudinary.com/dpspogkzf/image/upload/v1618869726/shoulditrain/logo_wl1ezx.svg) shoulditrain.today
 
-## Getting Started
+This project is based on a [quiz from a local (Northern VA, USA) nutrition company](https://www.instagram.com/p/CL7kiaqnP80/). Essentially, it is an 8 question quiz where the user must self-assess their answers. We then apply points to the answers that the user gives. At the end of the quiz, we perform a calculation and give a "% of readiness" for the user to expect from themselves.
 
-First, run the development server:
+This number is by no means definitive after taking the quiz consistently, it would give you a good baseline of how ready you feel to train on a given day.
```
Comment (zacjones93) [inline README.md:7]: The phrasing here is a little confusing.

-> This number is by no means definitive but after taking the quiz consistently you should see what a good baseline recovery is for yourself on any given day. This number will represent how ready you feel to train on that day

File: README.md [hunk 2]
```diff
@@ -1,38 +1,121 @@
...
+## Project Breakdown
+
+There are two main parts to this app:
+
+1. Capturing and storing user information
+2. Displaying the history of user information
+
+From a technical standpoint, this project demonstrates a few application state problems.
+
+The form itself has quite a bit of state. Theres user entered information you have to keep track of. Along with that, theres user progress threw a somewhat lengthy quiz that needs to be accounted for.
```
Comment (zacjones93) [inline README.md:30]:
```suggestion
The form itself has quite a bit of state. There's user-entered information you have to keep track of. Along with that, there's user progress through a somewhat lengthy quiz that needs to be accounted for.
```

**Review by zacjones93 — APPROVED** [review]
This looks great, had a couple of comments.

This long-form walk-through is great.

File: README.md (hunk 1 above)
Comment (theianjones) [inline README.md:7]:
```suggestion
This number is by no means definitive but after taking the quiz consistently you should see what a good baseline recovery is for yourself on any given day. This number will represent how ready you feel to train on that day
```

**Pull request merged**